frontend/lib/strapi-sdk/generated/sdk.ts

```diff
@@ -2667,118 +2667,6 @@ export type FindStoreQuery = {
                      } | null;
                   } | null;
                } | null;
-               menu3?: {
```

Comment:
This pull didn't intend to remove this I don't think. This was added in #1444 and I think accidentally removed here. @aburke07

Reply:
Ah! Thanks for finding this Danny!

Reply:
It caused a diff every time you ran a dev server, so I thought removing it was nice for DX

Reply:
This is autogenerated, so I wonder how it is possible that this was removed 🤔

Reply:
I think there was some confusion, this pull has more context: https://github.com/iFixit/react-commerce/pull/1455

Reply:
TLDR, nothing needs to change, menu3 shouldn't have been in the sdk in the first place